tensorflow/core/graph/gpu_fusion_pass.cc

```diff
@@ -427,9 +429,19 @@ class ROCmFusionOpAddNReluGrad : public ROCmFusionOpBase {
 
 //----------------------------------------------------------------------
 
+class ROCmFusionOpFMA : public ROCmFusionOpBase {
+ public:
+  ROCmFusionOpFMA(Graph* g) : ROCmFusionOpBase(g) {}
+
+ protected:
+  bool IsFusionEligible(const Node* n, FusionOpData* d) override;
+};
+
+//----------------------------------------------------------------------
+
 Status ROCmFusionPass::Run(const GraphOptimizationPassOptions& options) {
   // enable the fusion pass if the env var TF_ROCM_FUSION_ENABLE is set
-  if (ReadBoolFromEnvVar("TF_ROCM_FUSION_ENABLE")) {
+  if (!ReadBoolFromEnvVar("TF_ROCM_FUSION_DISABLE")) {
```

Comment:
this was coded to enable FMA fusion during the CI, however, I do want to have at least FMA on by default. Why wouldn't we want to enable all fusions by default?

Reply:
Fusions are disabled by default, because Conv* based fusions are not guranteed to work. We do not know the conv parameters during this optimization, and MIOpen Fusion API (which is used to implement those fusion) can and will fail to "compile" certain fusions (atleast that was the case 18 months ago, when we implemented this). So the decision was to disable fusion by default, and enable them manually for cases where we knew we will not run into MIOpen failures.

Reply:
Can we check with MIOpen to see if that is still true today?